### ISSUE TITLE
Access times of items?

### ISSUE THREAD

**Issue by HippoMan** [lastpass/lastpass-cli#110]
In the web interface, there's a column called "Last Touch" which at least approximates the last access time of the various items.

I don't see any way to get these same access times via "lpass".

Are they buried somewhere in one or more of the data structures used within that application, by any chance?

In any case, is there any chance that "lpass ls" or possibly other commands could be enhanced to return the last access time?

**Comment by bcopeland** [comment]
There is a ```skip(last_touch)``` in ```account_parse()``` in ```blob.c``` -- although I have not at all looked at what is in there, it may have what you seek.

**Comment by HippoMan** [comment]
Thank you.

Yes, that seems to be it. And there is also a last_modified_gmt. They seem to correspond to "access time" and "modification time" of the item.

What do you think about the following proposal? ...

What if I add an "--long" flag to the "lpass ls" command? If present, this flag would print these two timestamps, in addition to the other information which is listed.

It would be sort of like the difference between "ls"  and "ls -l".

I'm going to write a local version of that functionality, in order to test it. If you let me know that you're interested, I'll merge it into my branch and issue a pull.

**Comment by bcopeland** [comment]
Sure, that would be fine with me.

**Comment by HippoMan** [comment]
Thank you. I wrote and tested this code, and I have submitted it via a pull request.

I added a '--long' option with '-l' as its abbreviation to "lpass ls". This causes the last modification time to be listed, and also the last touch time, if it's available. These appear as follows:

[m: YYYY.MM.DD-hh:mm:ss]
[t: YYYY.MM.DD-hh:mm:ss]

I also describe these options in the usage message and the man page.
